lib/tasks/version.rake

```diff
@@ -0,0 +1,36 @@
+namespace :version do
+  namespace :vendor_api do
+    desc 'Displays VendorAPI versions across all environments'
+    task latest: :environment do
+      STDOUT.puts <<~LATEST_VERSIONS
+        =========================
+        Vendor API latest version
+        -------------------------
+        Production: v#{VendorAPI.production_version}
+        Sandbox: v#{VendorAPI::VERSION}
+        QA: v#{VendorAPI.development_version}
+        -------------------------
+      LATEST_VERSIONS
+    end
+
+    desc 'Display details of VendorAPI version change classes'
+    task :changes, %i[version] => :environment do |_, args|
+      specified_version = args.fetch(:version, nil)
+      VendorAPI.ordered_versions.each do |version, version_changes|
+        next if version.present? && specified_version != VendorAPI.version_number(version)
+
+        STDOUT.puts "Version v#{version}\n\n"
+        version_changes.each do |change_class|
```

Comment:
Presumably this is the order in which version changes are defined in `VendorAPI`? Is that preferable over alphabetical order? Wondering which is easier to scan but perhaps we should order them in `VendorAPI`, something to discuss with naming conventions.